connectivity/connectivity-demos/pom.xml

```diff
@@ -63,36 +63,6 @@
           </execution>
         </executions>
       </plugin>
-    </plugins>
-    <pluginManagement>
-      <plugins>
-        <!--This plugin's configuration is used to store Eclipse m2e settings only. It has no influence on the Maven build itself. -->
-        <plugin>
-          <groupId>org.eclipse.m2e</groupId>
-          <artifactId>lifecycle-mapping</artifactId>
-          <version>1.0.0</version>
-          <configuration>
-            <lifecycleMappingMetadata>
-              <pluginExecutions>
-                <pluginExecution>
-                  <pluginExecutionFilter>
-                    <groupId>org.apache.maven.plugins</groupId>
-                    <artifactId>maven-dependency-plugin</artifactId>
-                    <versionRange>[2.8,)</versionRange>
-                    <goals>
-                      <goal>copy-dependencies</goal>
-                    </goals>
-                  </pluginExecutionFilter>
-                  <action>
-                    <ignore />
-                  </action>
```

Comment:
we added this once; because we hand endless running maven-dependency-plugin copy executions in the designer ... you can see them in the console-view -> switching it to maven-console ... if this is no longer a problem, its good to run them.

Reply:
I could not reproduce the problem in Axon Ivy Designer.

Reply:
thanks for verifying though :+1: